pkg/reconciler/events/cloudevent/cloudevent.go

```diff
@@ -142,12 +156,31 @@ func eventForPipelineRun(pipelineRun *v1beta1.PipelineRun) (*cloudevents.Event,
 	return eventForObjectWithCondition(pipelineRun)
 }
 
+// eventForPipelineRun will create a new event based on a PipelineRun,
+// or return an error if not possible.
+func eventForRun(run *v1alpha1.Run) (*cloudevents.Event, error) {
```

Comment:
method comment should read `new event based on a Run` and `check if the Run is defined`

Reply:
+1